src/pages/en/guides/images.md

```diff
@@ -193,15 +199,27 @@ This component is useful to optimize what your user sees at various screen sizes
 Check out MDN's guide for more information about [responsive images and art direction](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#art_direction).
```

Comment:
Looking at this now, I don't think we need to link to the same article, twice in a row. If that original link is what we'd use to describe sizes, then I think it's sufficient just to mention it once. If they need "help" (with anything, `sizes` or otherwise), we've given them that article as a reference.